packages/core/test/testing-library.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { render, RenderResult } from '@testing-library/react'
+import { ThemeProvider } from '../src'
+
+const WithThemeProvider = ({ children }) => (
+  <ThemeProvider>{children}</ThemeProvider>
+)
+WithThemeProvider.propTypes = {
+  children: PropTypes.node,
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
same thing here. seems silly to update this file from JS to TS if all we are doing is adding noise.